packages/web/src/components/layouts/HomeNew/HomeNew.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Hero from '../../ui/Hero/Hero';
+import Container from 'components/ui/Container/Container';
+import CardList from '../../ui/CardList/CardList';
+import CardGroups from '../../ui/CardGroups/CardGroups';
+import Footer from '../../ui/Footer/Footer';
+import CardGroup from '../../ui/CardGroup/CardGroup';
+
+export type HomeNewProps = {
```

Comment:
For a single component inside a single file I would rather use `Props` name always without an export. The naming would be simpler and we should avoid importing types from components.

```ts
type Props = { /* Component props */ }

export default function Component(props: Props) {
  // ...
}
```